core/persistence-api/src/main/java/org/apache/syncope/core/persistence/api/dao/search/SearchCond.java

```diff
@@ -275,7 +275,7 @@ public String hasAnyTypeCond() {
         String anyTypeName = null;
 
         if (type == null) {
-            return anyTypeName;
+            return null;
```

Comment:
What's the rationale behind such change?

Reply:
Original variable name is always `null` here. It's best to be explicit about what is returned, rather than letting the compiler dereference later. Makes it difficult to read.

Reply:
Hmm,  not sure about this, but also no strong opinion.
Let's leave it as per your change.